packages/client/src/intros/shared/Intro.tsx

```diff
@@ -28,22 +29,21 @@ export default ({
   usableForBullets = [],
   usableForText,
 }: Props) => {
+  const { t } = useTranslation();
   const influence =
     dependantOnSituation && dependantOnQuestions
-      ? "Uw situatie en uw antwoorden bepalen of u een omgevingsvergunning nodig hebt."
+      ? t("introPage.common.first situation")
```

Comment:
Can you change the key to `introPage.common.situation dependent on both situation and questions`?

Reply:
I added a subject to the dev meeting to discuss this. I think we should have a convention to do it all in english, 
Have descriptive names.  For example first situation, is probably not always gone be the first, maybe it will change.

But also not too long. 

i like you suggestion, let's use that one.  :-)